battle/attack/ContWaveDef.java

```diff
@@ -51,10 +51,10 @@ public void update() {
 		}
 		if (!activate)
 			return;
-		if (t == (isMini ? W_MINI_TIME : W_TIME)) {
+		if (t == (isMini ? W_MINI_TIME - 1 : W_TIME)) {

```

Comment:
I want to know the reason of -1 in line 54. If it's about adding inverted wave, you shouldn't touch other things

Reply:
It was shown as a conflict in the GitHub summary.